doc/releases/changelog-dev.md

```diff
@@ -324,9 +328,6 @@
   [(#6278)](https://github.com/PennyLaneAI/pennylane/pull/6278)
   [(#6310)](https://github.com/PennyLaneAI/pennylane/pull/6310)
 
-* Fixes a bug where `None` was added to the wires in `qml.PhaseAdder`, `qml.Adder` and `qml.OutAdder`.
-  [(#6360)](https://github.com/PennyLaneAI/pennylane/pull/6360)
-
```

Comment:
is this meant to be deleted?

Reply:
Weird ... I don't remember doing that. Let me undo it. 😓

Reply:
@lillian542 Addressed in the latest commit.